roles/edpm_neutron_metadata/defaults/main.yml

```diff
@@ -57,9 +57,11 @@ edpm_neutron_metadata_agent_ovn_ovsdb_probe_interval: '60000'
 # tls
 edpm_neutron_metadata_agent_tls_enabled: "{{ edpm_tls_certs_enabled | default(False) }}"
 
+edpm_neutron_metadata_tls_cacert_bundle_src: "/var/lib/openstack/cacerts/{{ edpm_neutron_metadata_service_name }}/tls-ca-bundle.pem"
+edpm_neutron_metadata_tls_cacert_bundle_dest: "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+edpm_neutron_metadata_tls_cacert_volumes: []
```

Comment:
Default in argspecs and in defaults/main.yml must match. You can add note that it's set in as part of the configuration.